plugins/lsp/language-servers/default.nix

```diff
@@ -700,8 +701,14 @@ in
     let
       mkLsp = import ./_mk-lsp.nix;
       lspModules = map mkLsp servers;
+      baseLspPath = [ "plugins" "lsp" "servers" ];
+      renamedServers = {
+        tsserver = "ts-ls";
+      };
```

Comment:
It's kinda confusing having two `let` blocks in this file, but I think this would make more sense in the first `let` block? As in directly after the `servers` list.

That way the first let block is purely declarative while this one is purely implementation.